hoodie-client/src/main/java/com/uber/hoodie/io/compact/HoodieRealtimeTableCompactor.java

```diff
@@ -155,14 +156,15 @@ private boolean isCompactionSucceeded(HoodieCompactionMetadata result) {
                             HoodieTimeline.DELTA_COMMIT_ACTION))
             .filterCompletedInstants().lastInstant().get().getTimestamp();
 
-    HoodieCompactedLogRecordScanner scanner = new HoodieCompactedLogRecordScanner(fs, operation.getDeltaFilePaths(), readerSchema, maxInstantTime);
+    HoodieCompactedLogRecordScanner scanner = new HoodieCompactedLogRecordScanner(fs, metaClient.getBasePath(),
```

Comment:
whitespace change?

Reply:
`metaClient.getBasePath()` is new argument added.